compiler/tests/Tests/Syntax.hs

```diff
@@ -170,8 +170,10 @@ verifySchemaDef baseName schemaName =
         (Bond _ _ declarations) <- parseBondFile [] $ "tests" </> "schema" </> baseName <.> "bond"
         let schema = fromJust $ find ((schemaName ==) . declName) declarations
         return $
-            -- some versions aeson encode angle brackets
             BL.fromStrict $
+            -- aeson-pretty-print prints anything above 1e19 in scientific notation
+            replace "1.8446744073709551615e19" "18446744073709551615" $
```

Comment:
In theory, I should be able to use [Custom Constructor](http://hackage.haskell.org/package/aeson-pretty-0.8.7/docs/Data-Aeson-Encode-Pretty.html#t:NumberFormat) but this seems simpler change for the test.
I can make the change, if the ghc installation work around is acceptable.

Reply:
cc: @chwarr  and @lalo 

Do you have any suggestions on the right way to go about this? I will be happy to make the change that way.


reference:  NumberFormat pretty printing in aeson-pretty http://hackage.haskell.org/package/aeson-pretty-0.8.7/docs/Data-Aeson-Encode-Pretty.html#t:NumberFormat
Standard generated file: https://github.com/microsoft/bond/blob/3132eb2b90d29d916e703e42ec6482b9b89a86ce/compiler/tests/generated/schemadef/defaults.Foo.json#L249

Reply:
What you have here is fine by me. Both are valid JSON representations of the same number.